### PR TITLE
add logo macro for localised logo

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

File: cms/templates/logo.pt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal"
+     metal:define-macro="logo">
+    <div id="logo-container" tal:define="localisation view.get_localisation()" tal:condition="localisation">
+        <a href="/" tal:condition="localisation.logo_image">
```
Comment (miltontony) [inline cms/templates/logo.pt]: We'll need a default logo. `/static/img/logo.png` should suffice

Comment (Rizziepit) [inline cms/templates/logo.pt]: Is there a case where you'd want no logo image, only logo text?

File: cms/templates/logo.pt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+            <img src="${view.get_image_url(localisation.logo_image_host, localisation.logo_image)}" border="0" alt="${localisation.logo_description}"/>
+        </a>
+        <div tal:condition="localisation.logo_text">${localisation.logo_text}</div>
```
Comment (miltontony) [inline cms/templates/logo.pt]: no, `logo_text` is a special case for MAMA at the moment. I'd leave it out from here and implement it on `unicore-cms-mama` override only.

Comment (Rizziepit) [inline cms/templates/logo.pt]: Ok. How about `default_image_src` argument for the macro? If it's left out, nothing is rendered. Apps can then specify `default_image_src="/static/img/logo.png"` if they want.

Comment (miltontony) [inline cms/templates/logo.pt]: +1 on that

**Comment by Rizziepit** [comment]
@miltontony I updated the logo macro so that you can use it like this in a template:
```html
<div metal:use-macro="view.logo_template" tal:define="default_image_src '/default/path/to/logo.png'"></div>
```
Also added a test

**Comment by Rizziepit** [comment]
@miltontony can I merge and make a new release?

**Comment by smn** [comment]
Do we want a test when we fallback to the default image?

File: cms/templates/logo.pt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal"
+     metal:define-macro="logo">
+    <?python
+        try:
+            img_attrs
+        except NameError:
+            img_attrs = view.get_logo_attributes()
+    ?>
```
Comment (smn) [inline cms/templates/logo.pt]: bleeding

Comment (smn) [inline cms/templates/logo.pt]: I know the Chameleon story here, don't worry.

File: cms/templates/logo.pt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <div id="logo-container" tal:condition="img_attrs['src']">
+        <a href="/">
+            <img tal:attributes="img_attrs" border="0"/>
+        </a>
+    </div>
+</div>
```
Comment (smn) [inline cms/templates/logo.pt]: Missing new line.

**Comment by smn** [comment]
:+1: in general and let's fast track moving away from Chameleon.

**Pull request closed without merging**